.github/workflows/pipeline.yml

```diff
@@ -125,80 +179,32 @@ jobs:
-        run: node script/release-make-skeleton.js
-
-      - name: Download linux artifacts
-        uses: actions/download-artifact@master
+      - name: Download release
+        uses: actions/download-artifact@v2
         with:
-          name: ubuntu-latest
-          path: _release/platform-linux-x64
-
-      - name: Download macOS artifacts
-        uses: actions/download-artifact@master
-        with:
-          name: macos-latest
-          path: _release/platform-darwin-x64
-
-      - name: Download windows artifacts
-        uses: actions/download-artifact@master
-        with:
-          name: windows-latest
-          path: _release/platform-windows-x64
-
-      - name: Create archives for Github release
-        run: |
-          cd _release/platform-darwin-x64
-          zip -r ../../tailwind-ppx-darwin-x64.zip .
-          cd ../..
-
-          cd _release/platform-linux-x64
-          zip -r ../../tailwind-ppx-linux-x64.zip .
-          cd ../..
-
-          cd _release/platform-windows-x64
-          zip -r ../../tailwind-ppx-windows-x64.zip .
-          cd ../..
-      
+          name: release
+          path: release
+      - name: Zip release folder
+        run: zip -r release.zip release
       - name: Create GitHub release
         id: create_release
-        uses: actions/create-release@v1.0.0
+        uses: actions/create-release@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
         with:
           tag_name: ${{ github.ref }}
           release_name: Release ${{ github.ref }}
           draft: false
           prerelease: false
-        
-      - name: Upload tailwind-ppx-darwin-x64.zip to Github release 
-        uses: actions/upload-release-asset@v1.0.1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
-        with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: tailwind-ppx-darwin-x64.zip
-          asset_name: tailwind-ppx-darwin-x64.zip
-          asset_content_type: application/gzip
-
-      - name: Upload tailwind-ppx-linux-x64.zip to Github release 
-        uses: actions/upload-release-asset@v1.0.1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
-        with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: tailwind-ppx-linux-x64.zip
-          asset_name: tailwind-ppx-linux-x64.zip
-          asset_content_type: application/gzip
 
-      - name: Upload tailwind-ppx-windows-x64.zip to Github release 
-        uses: actions/upload-release-asset@v1.0.1
+      - name: Upload ubuntu-latest to Github release
+        uses: actions/upload-release-asset@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
         with:
           upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: tailwind-ppx-windows-x64.zip
-          asset_name: tailwind-ppx-windows-x64.zip
+          asset_path: release.zip
+          asset_name: release.zip
```

Comment:
These changes aren't super clear to me --- will both the MacOS and Windows binaries still be published to the GitHub release?

Reply:
What will be published is a zip file `release.zip` which contains artifacts for all platforms and a `postinstall.js` + some other stuff. 

You can download the `release.zip` file and check what it contains here: https://github.com/dylanirlbeck/tailwind-ppx/actions/runs/128834698

Reply:
Gotcha, that makes sense.